File: python/oop/SOLID/bird/Parrot.py
from Bird import *
from BirdType import *

class Parrot(Bird):
	def __init__(self, weight, colour, size, beakType, birdType):
		super().__init__(weight, colour, size, beakType, birdType)

	def fly(self):
		print("Parrot is flying")

File: oop/code/oop/src/main/java/com/scaler/lld/bird/simple/Parrot.java
package com.scaler.lld.bird.simple;

public class Parrot extends Bird {

    public Parrot(Double weight, String colour, String size, String beakType, BirdType type) {
        super(weight, colour, size, beakType, type);
    }

    @Override
    public void fly() {
        System.out.println("\nParrot is flying");
    }
    
}
